### DISCUSSION TITLE
How to pass the "-Dsolr.disable.allowUrls=true" configuration to the Solr container while using testcontainers library for executing tests

### DISCUSSION THREAD

**Discussion by MUsman1992** [testcontainers/testcontainers-java#6902] [Q&A]
On my system, when I run Solr in the Docker Container, I use this command

`docker run -p 8983:8983 -v $PWD/:/var/solr/data -v $PWD/configsets:/opt/solr-9.1.1/server/solr/configsets/ -t solr -a "-Dsolr.disable.allowUrls=true"    `

I am now using testcontainers to deploy Solr Container while executing tests. Now I am unable to figure out how to pass "-Dsolr.disable.allowUrls=true"   configuration to the container. I have spent couple of hours on this issue but still I am unable to figure it out. Can anyone guide me in this regard? Which function of the testcontainers library can i use for this purpose? Thanks

**Comment by eddumelendez** [comment]
I think you can use `.withEnv("SOLR_OPTS", "-Dsolr.disable.allowUrls=true")`.

**Comment by MUsman1992** [comment]
@eddumelendez now I am getting this error after using your function any idea how to fix this
![image](https://user-images.githubusercontent.com/62036653/230491876-a6a26461-7959-46f5-b6a1-1dd65e4481a2.png)

**Reply by eddumelendez** [reply]
sorry, not a solr expert 😕
